src/main/java/com/accelad/acctive/sim/kernel/core/evaluator/diff/BetweenStrictlyLessThanMaxFunction.java

```diff
@@ -16,7 +16,7 @@
     private final DifferentialFunction<X> firstBound;
     private final DifferentialFunction<X> secondBound;
 
-    public BetweenFunction(MathFactory<X> DFFactory, DifferentialFunction<X> x,
+    public BetweenStrictlyLessThanMaxFunction(MathFactory<X> DFFactory, DifferentialFunction<X> x,
```

Comment:
Name looks weird. It seems to mention only one side of the inequality test.
max(a, b) > v >= min(a, b)